backend/connection.go

```diff
@@ -18,9 +18,12 @@ const (
 	// many lines to tail from.
 	bytesToLines int64 = 120
 
-	// defaultTailLines is the number of lines to tail by default if the value
-	// is not overriden.
-	defaultTailLines int64 = 30
+	// defaultTailLines is the number of lines to tail by default.
+	defaultTailLines int64 = 1000
```

Comment:
yeah, too much? I don't think it will overwhelm the browser and this way you do get a nice piece of history up-front.